src/main/java/frc/robot/subsystems/ClawSubsystem.java

```diff
@@ -15,12 +16,30 @@ public class ClawSubsystem extends SubsystemBase {
   public ClawSubsystem() {}
 
   private final CANSparkMax motleft = new CANSparkMax(14, MotorType.kBrushless);
-  private final CANSparkMax motright = new CANSparkMax(13, MotorType.kBrushed);
-  private final DutyCycleEncoder leftenc = new DutyCycleEncoder(0);
-  private final DutyCycleEncoder rightenc = new DutyCycleEncoder(1);
+  private final CANSparkMax motright = new CANSparkMax(13, MotorType.kBrushless);
+  private final RelativeEncoder leftenc = motleft.getEncoder();
+  private final RelativeEncoder rightenc = motright.getEncoder();
 
   MotorControllerGroup clawGroup = new MotorControllerGroup(motleft, motright);
 
+  public void zeroclaw() {
```

Comment:
Leave a comment that the claw subsystem expects claws to start in the full open position when zeroed.